### PR TITLE
Gp 1190

### PR DESCRIPTION
**Pull request by mlaszloffy**

### CODE REVIEW

File: seqware-admin-webservice/src/main/java/io/seqware/webservice/controller/CustomWorkflowRunFacadeREST.java [hunk 1]
```diff
@@ -199,8 +199,8 @@ public void deleteRecursive(@PathParam("id") Integer id, Set<ModelAccessionIDTup
             return null;
         }
         // if the workflow run is not in a settled state, then abort
-        if (!(data.getStatus().equals(WorkflowRunStatus.completed.name()) || data.getStatus().equals(WorkflowRunStatus.failed.name()) || data
-                .getStatus().equals(WorkflowRunStatus.cancelled.name()))) {
+        if (!(data.getStatus() == null || data.getStatus().isEmpty() || data.getStatus().equals(WorkflowRunStatus.completed.name())
```
Comment (morgantaschuk) [inline seqware-admin-webservice/src/main/java/io/seqware/webservice/controller/CustomWorkflowRunFacadeREST.java]: Are you sure `data` will never be null?

File: seqware-ext-admin-testing/src/test/java/net/sourceforge/seqware/pipeline/plugins/DeletionDBIT.java [hunk 2]
```diff
@@ -33,7 +33,7 @@
  * 
  * @author dyuen
  */
-public class DeleteDBET {
+public class DeletionDBIT {
```
Comment (morgantaschuk) [inline seqware-ext-admin-testing/src/test/java/net/sourceforge/seqware/pipeline/plugins/DeletionDBIT.java]: What does "I" stand for?

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

File: seqware-admin-webservice/src/main/java/io/seqware/webservice/controller/CustomWorkflowRunFacadeREST.java (hunk 1 above)
Comment (mlaszloffy) [inline seqware-admin-webservice/src/main/java/io/seqware/webservice/controller/CustomWorkflowRunFacadeREST.java]: Check for null is performed above.

File: seqware-ext-admin-testing/src/test/java/net/sourceforge/seqware/pipeline/plugins/DeletionDBIT.java (hunk 2 above)
Comment (mlaszloffy) [inline seqware-ext-admin-testing/src/test/java/net/sourceforge/seqware/pipeline/plugins/DeletionDBIT.java]: Integration(Test)

**Comment by mlaszloffy** [comment]
retest this please

**Pull request merged**